app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,65 @@
+<section>
+  <h2><%= @recipe.name %></h2>
+
+  <div>
+    <div>
+      <% if @recipe.public == false %>
+        <%= check_box :public, checked: false  %>
+        <label class="check-label" for="unchecked">Public</label>
+      <% else %>
+        <%= check_box :public, checked: true %>
+        <label class="check-label" for="checked">Public</label>
+      <%end%>
+    </div>
+
+    <div>
+      <p>Preparation time: </p>
+      <span><%=@recipe.preparation_time %></span>
+    </div>
+
+    <div>
+      <p>Cooking time: </p>
+      <span><%=@recipe.cooking_time %></span>
+    </div>
+
+    <div>
+      <p><%= @recipe.description %></p>
+    </div>
+
+    <div>
+      <div>
+        <%= link_to 'Generate shopping list', recipes_path %>
```

Comment:
Kindly check; the path needs to match the general shopping list path. 
You can confirm the path using `rails routes`

Reply:
Hey, @principlebrothers I feel it's quite unnecessary for this now as the requirements on the kanban have been met. This can be dealt with when we are working on the shopping list

Reply:
@DuaneDave The route can be dealt with like you suggested however, kindly remember to add the authorization.